pkg/kmm/kernelmapping.go

```diff
@@ -21,44 +21,28 @@ type KernelMappingBuilder struct {
 // KernelMappingAdditionalOptions additional options for KernelMapping object.
 type KernelMappingAdditionalOptions func(builder *KernelMappingBuilder) (*KernelMappingBuilder, error)
 
+func NewKernelMappingBuilder() *KernelMappingBuilder {
```

Comment:
Can you create object with empty kernel mapping?

Reply:
Creating an empty kernelMapping means that all the values in the KernelMapping object are getting the go default value, pointers are getting `nil`, strings are getting `""` etc.
Therefore, an empty kernel mapping means that `containerImage==""`. KMM's validation webhook won't allow it.

In order to test that validation webhook, we should be able to create an empty kernelMapping in the *tests* to validate that the webhook is indeed working as expected.